cmd/estuary-shuttle/main.go

```diff
@@ -1381,6 +1383,9 @@ func (s *Shuttle) loadCar(ctx context.Context, bs blockstore.Blockstore, r io.Re
 	return car.LoadCar(ctx, bs, r)
 }
 
+func createRetrievalURL(cid string) string {
```

Comment:
move this method into the `util.content` package and use in main and shuttle main